domains/client/domain-executor/src/tests.rs

```diff
@@ -80,10 +81,194 @@ async fn test_executor_full_node_catching_up() {
     );
 }
 
-// TODO: enable the test after we can fetch call_data from the original primary block, currently the test
-// will fail due to `panicked at 'Bad input data provided to initialize_block_with_post_state_root: Codec error'`
 #[substrate_test_utils::test(flavor = "multi_thread")]
 #[ignore]
+async fn test_initialize_block_proof_creation_and_verification_should_work() {
+    test_invalid_state_transition_proof_creation_and_verification(0).await
+}
+
+#[substrate_test_utils::test(flavor = "multi_thread")]
+#[ignore]
+async fn test_apply_extrinsic_proof_creation_and_verification_should_work() {
+    test_invalid_state_transition_proof_creation_and_verification(1).await
+}
+
+#[substrate_test_utils::test(flavor = "multi_thread")]
+#[ignore]
+async fn test_finalize_block_proof_creation_and_verification_should_work() {
+    test_invalid_state_transition_proof_creation_and_verification(2).await
+}
```

Comment:
When will these tests be enabled? What features to enable them are still missing?

Reply:
When we enable fraud proof here:
https://github.com/subspace/subspace/blob/1ab1eab738cf7a73f1d3c127d8e0b61c9e1904de/domains/client/domain-executor/src/domain_block_processor.rs#L385-L387

It seems we are good to proceed with this step as https://github.com/subspace/subspace/issues/1230 is completed, right?

Reply:
Yes, it's time to enable the fraud proof again, especially with these tests added now.